python/eos/parameter_TEST.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import eos
+import numpy as np
+import yaml
+
+class ClassMethodTests(unittest.TestCase):
+
+    def test_importing_from_wcxf(self):
+
+        from wilson import Wilson
+
+        inputs = {
+	    'cbtaunutau::cSL':  0.3 + 0.1j,
+            'ubenue::cVL':     -0.1 + 0.2j,
+        }
+        w = Wilson(inputs, scale=4.2, eft="WET", basis="EOS")
+        p = eos.Parameters.FromWCxf(w)
+
```

Comment:
Can we check the values of `p`, to ensure correct import? This is more for future updates of the code...

Reply:
Yes. To be kept in mind: Wilson/WCxf parametrizes the BSM contributions. We still have to add the SM contributions on top of that.